web/src/user_group_create.ts

```diff
@@ -240,7 +243,7 @@ export function set_up_handlers(): void {
     });
 
     const $pill_container = $container.find(".can-manage-group-container .pill-container");
-    settings_components.create_group_setting_widget({
+    can_manage_group_widget = settings_components.create_group_setting_widget({
```

Comment:
There is no need to create the widget each time opening the creation form. We would anyway need to have a variable like `can_manage_group_widget` to get the value.

Reply:
This seems OK as long as it doesn't create live-update issues with the widget having stale values. I guess there is also a (very small) web app startup cost to making potentially unused widgets.

Reply:
There is no new live-update bug, as there is no code changes that would lead to it.

But a couple of things like which were present before this PR like and we should probably fix-
- Going to group edit UI from creation form and then coming again to the creation form retains the setting values selected.
- Opening the creation form after creating a group successfully will also have the same setting values that were selected while creating the previous group. 

These could be fixed either by just setting the widget to defaul value each time we open the creation form or just recreating the whole widget each time.